src/Validators/validateBusinessName.ts

```diff
@@ -0,0 +1,9 @@
+import { EmptyNameError, InvalidBusinessName } from "../Errors";
+
+const validateBusinessName = businessName => {
+  const regex = /^((?![\^!@#$*~ <>?]).)((?![\^!@#$*~<>?]).){0,73}((?![\^!@#$*~ <>?]).)$/;
```

Comment:
are we sure? creo que es muy preferible el riesgo de que una empresa llene una razon social invalida a decirle "razon social invalida" a algo que resultaba ser valido solo porque pintó validar de más